deploy/tiller.jsonnet

```diff
@@ -22,6 +23,7 @@
                 }
             },
             "spec": {
+                "automountServiceAccountToken": false,
```

Comment:
This line seems like an upstream bug to me - and sure enough I see kubernetes/helm#3985
I suspect if you tested with this particular change, you would find that tiller was unable to talk to k8s.